client/components/sectionform/sectionform.controller.js

```diff
@@ -0,0 +1,69 @@
+angular.module('venueApp')
+  .controller('SectionFormController', ($scope, $location, $routeParams, Auth, Course, Section) => {
+
+    Auth.getCurrentUser((user) => {
+      $scope.user = user;
+      Course.get({
+        id: $routeParams.id,
+        withSections:true
+      }, course => {
+        $scope.course = course;
+        if($scope.updating == "true"){
+          setCurrentSection();
+        }
+      });
+    });
+
+    function setCurrentSection(){
+      var currentSection;
+      $scope.course.sections.forEach((section, index)=>{
+        if(section._id == $routeParams.sectionId){
+          currentSection = index;
+        }
+      });
+      $scope.section = $scope.course.sections.splice(currentSection, 1)[0];
+      $scope.section.sectionNumbers = $scope.section.sectionNumbers.toString();
+      $scope.isInstructor = $scope.section.instructors.some((instr)=>{
+        return instr == $scope.user._id;
+      })
```

Comment:
:penguin: missing semicolon, also array should be made a boolean via `.length > 0` (it's just kind of weird)